### PR TITLE
More Examples: InCodeTexTemplateExample and further additions

### PR DESCRIPTION
**Pull request by kolibril13**
More examples for the docs

### CODE REVIEW

**Comment by kolibril13** [comment]
Does someone know what this means:
```
                    INFO     Rendered Example1Text                   scene.py:93
                             Played 0 animations                                
                    INFO     Writing "\mathbb{M}" to /hom tex_file_writing.py:81
                             e/docs/checkouts/readthedocs                       
                             .org/user_builds/manimce/che                       
                             ckouts/547/docs/source/media                       
                             /tex/f483963d3a30f7b8.tex                          
unknown option --pdf
```
@cobordism ?
But for me it would be also fine when we remove this coustomtex example from the docs.

**Comment by cobordism** [comment]
I'll look into it. I'll get back to you here.

**Comment by cobordism** [comment]
So, here is how it works.

Some tex files need to be compiled to pdf (instead of dvi or xdv) in order to render correctly.
We use dvisvgm to convert the output into svg graphics. 
The `--pdf` flag is passed to dvisvgm to convert a pdf to an svg.

Can you type `dvisvgm --help` on your terminal and paste the output here? 
```
$ dvisvgm --help | grep pdf
  -P, --pdf                     convert PDF file to SVG
```

**Comment by kolibril13** [comment]
It gives me this:
```
$ dvisvgm --help | grep pdf
       dvisvgm --pdf [options] pdffile
  -P, --pdf                     convert PDF file to SVG

```

**Comment by cobordism** [comment]
so what's with the "unknown option --pdf" ??
I don't get it

**Comment by kolibril13** [comment]
The docs build fails on this pr, see here:
https://readthedocs.org/projects/manimce/builds/12108015/
And it is because of the Example1Text that I added in this pr.
So in case that we want to show Example1Text in the docs, I was thinking if you know why the docs build fails.

**Comment by cobordism** [comment]
let's take this discussion to discord.

**Comment by kolibril13** [comment]
Hmm, also with merging the current master, the docs are still not building:

**Comment by behackl** [comment]
#554 only fixed a formatting issue, you run into the problem that the `dvisvgm` version provided by readthedocs is too old. Even if we would ask them to switch to their latest docker image (which requires manual interaction with them), we still have

```
docs@e08634e2e1c3:~$ dvisvgm --version
dvisvgm 2.1.3
```

which is too old:

```
docs@e08634e2e1c3:~$ dvisvgm --pdf
unknown option --pdf
```

If there is not a very, very, very simple way which allows to upgrade the version of `dvisvgm`, I don't really see much hope for getting this to work within the current infrastructure.

Do you really need to use pdflatex in this particular example? Or did you just want to illustrate how to use it? If so, we might have to simply document it in the reference manual without actually rendering a video.

**Comment by cobordism** [comment]
> Do you really need to use pdflatex in this particular example? Or did you just want to illustrate how to use it? If so, we might have to simply document it in the reference manual without actually rendering a video.

This exampled does not require pdf. I was just illustrating the use.
I have run into other examples that would not render correctly when using dvi/xdv, but this is not one of them.

**Comment by cobordism** [comment]
Readthedocs tells me:

"the image with ubuntu 20.04 isn't available yet, the latest testing image has ubuntu 18.04. We may release the new one in a couple of weeks"

**Comment by kolibril13** [comment]
ok, nice!
Then I will close this pr for now, and maybe in a few weeks will come back to it.

File: docs/source/examples/shapes_images_positions.rst [hunk 1]
```diff
@@ -16,6 +16,18 @@ Shapes, Images and Positions
             self.add(d, c, s, t)
             self.wait(1)
 
+.. manim:: InterpolationExample
```
Comment (kolibril13) [inline docs/source/examples/shapes_images_positions.rst:19]:
```suggestion

```

File: docs/source/examples/shapes_images_positions.rst [hunk 2]
```diff
@@ -16,6 +16,18 @@ Shapes, Images and Positions
...
+
+    class InterpolationExample(Scene):
+        def construct(self):
+            dot = Dot()
+            dot2 = Dot().shift(LEFT)
+            dot3 = VMobject().interpolate(dot, dot2, alpha=0.4)
+            self.add(dot, dot2, dot3)
+            self.wait(2)
+
+
```
Comment (kolibril13) [inline docs/source/examples/shapes_images_positions.rst:30]:
```suggestion

```

**Pull request closed without merging**